tests/distributed/test_dist_graph_store.py

```diff
@@ -76,10 +76,21 @@ def run_client(graph_name, barrier, num_nodes, num_edges):
     assert g.node_attr_schemes()['test1'].dtype == F.int32
     assert g.node_attr_schemes()['features'].shape == (1,)
 
+    selected_nodes = np.random.randint(0, 100, size=g.number_of_nodes()) > 30
+    # Test node split
+    local_nids = F.nonzero_1d(g._g.ndata['local_node'])
+    local_nids = g._g.ndata[dgl.NID][local_nids]
```

Comment:
Can we avoid using `g._g` in unittests? We should treat us as users when writing unittests.